autopatches/patches/missing-seteu-in-change_url/patch.sh

```diff
@@ -0,0 +1,10 @@
+
+cd scripts/
+
+if [ ! -e change_url ] || grep -q 'ynh_abort_if_errors' change_url
+then
+    # The app doesn't has any change url script or already has ynh_abort_if_error
+    exit 0
+fi
+
+sed 's@\(source /usr/share/yunohost/helpers\)@\1\nynh_abort_if_errors@g' -i change_url
```

Comment:
we should probably add a backup/restore mechanism right before `ynh_abort_if_errors`

Reply:
Zblerg yeah I saw that on several apps and in fact I don't understand why it's done this way ... Shouldn't we just apply set -eu as quick as possible at the beginning of the script ...?

(In fact there's a similar weird stuff in upgrade script were we patch "backward compatibility stuff" *before* the safety backup ... but imho the point of the safety backup is precisely to save the app state before any modification starts to be applied...)